qiwis.py

```diff
@@ -248,24 +248,25 @@ def load(self, appInfos: Mapping[str, AppInfo]):
             self.createApp(name, info)
         logger.info("Loaded %d app(s)", len(appInfos))
 
-    def addFrame(self, name: str, frame: QWidget, info: AppInfo):
+    def addFrame(self, name: str, title: str, frame: QWidget, info: AppInfo):
         """Adds the given frame and wraps it with a wrapper widget.
 
         This is not a qiwiscall because QWidget is not Serializable.
         
         Args:
-            name: A name of app.
-            frame: A frame to show.
-            info: An AppInfo object describing the app.
+            name: The app name to which frame is added.
+            titme: The frame title.
+            frame: Frame object to show.

```

Comment:
Since we didn't define `Frame` as a class, this expression looks awkward a little.
How about e.g. `Frame widget to show.`?

Reply:
Oh, it looks more reasonable!